File: зад3.java
import java.util.Scanner;
public class Main{
    
 public static int Re(int number) {
    int result = 0;
    //Переворачиваем
    while (number != 0) {
        result = result * 10 + (number % 10);
        //Деление без остатка
        number= number / 10;
    }
    return result;
 }
 
public static void main(String[] args) {
    //можно использовать созданный объект scanner для чтения данных.
    Scanner scanner = new Scanner(System.in);
    System.out.print("Введите количество: ");
    int n = scanner.nextInt();
    //Ввод чисел
    for (int i = 0; i < n; i++) {
         System.out.print("\nВведите число: ");
        int number = scanner.nextInt();
        //Преравниваем
        number=Re(number);
        System.out.print(number);
    }
   }
}

File: зад3.kt
import java.util.Scanner 
fun Re(number: Int): Int { 
    var result=0
    var x = number
    while (x != 0) {
        result = result * 10 + (x % 10);
        //Деление без остатка
        x= x / 10;
    }
    return result;
} 
fun main() { 
   print("enter quantity: ") 
   val scanner = Scanner(System.`in`) 
   val n = scanner.nextInt()  
    for (i in 0..n-1) { 
        print("\nenter number: ")
        //val scanner = Scanner(System.`in`) 
        var number = scanner.nextInt()  
        number=Re(number)
        print(number)
    } 
   scanner.close(); 
}
